operations/helm/charts/mimir-distributed/templates/_helpers.tpl

```diff
@@ -742,3 +782,57 @@ kubectl image reference
 {{- define "mimir.kubectlImage" -}}
 {{ .Values.kubectlImage.repository }}:{{ .Values.kubectlImage.tag }}
 {{- end -}}
+
+{{/*
+TLS configuration template for internal Mimir components
+*/}}
+{{- define "mimir.serverTlsConfig" -}}
```

Comment:
do we also want to change the `mimir.gatewayUrl`, `mimir.remoteWriteUrl.inCluster` and `mimir.remoteReadUrl.inCluster` tempaltes?